test/docs/test_tutorial.py

```diff
@@ -96,19 +95,19 @@ def test_circuit_builder_qft() -> None:
 qubit[5] q
 
 H q[0]
-CRk(2) q[1], q[0]
-CRk(3) q[2], q[0]
-CRk(4) q[3], q[0]
-CRk(5) q[4], q[0]
+CRk q[1], q[0], q[2]
+CRk q[2], q[0], q[3]
+CRk q[3], q[0], q[4]
+CRk q[4], q[0], q[5]
 H q[1]
-CRk(2) q[2], q[1]
-CRk(3) q[3], q[1]
-CRk(4) q[4], q[1]
+CRk q[2], q[1], q[2]
+CRk q[3], q[1], q[3]
+CRk q[4], q[1], q[4]
 H q[2]
-CRk(2) q[3], q[2]
-CRk(3) q[4], q[2]
+CRk q[3], q[2], q[2]
+CRk q[4], q[2], q[3]
 H q[3]
-CRk(2) q[4], q[3]
+CRk q[4], q[3], q[2]
```

Comment:
What happened here? CRk is a gate with an angle param, of float type, and two qubit operands.